src/telemetry/http/mod.rs

```diff
@@ -1,17 +1,30 @@
 use std::fmt;
-use std::time::Duration;
+use std::time::{Duration, Instant};
 
-use super::{
+use super::metrics::{
     latency,
-    prom::FmtMetrics,
     Counter,
+    FmtMetrics,
     Histogram,
-    RequestLabels,
-    ResponseLabels,
     Scopes,
-    Stamped,
 };
 
+pub mod event;
+mod labels;
+mod record;
+mod sensors;
+pub mod service;
+
+pub use self::labels::{RequestLabels, ResponseLabels};
+pub use self::record::Record;
+pub use self::sensors::Sensors;
+
+#[derive(Debug)]
+pub(super) struct Stamped<T> {
```

Comment:
Out of curiosity: I'm taking this to mean we don't intend to start expiring non-HTTP metrics any time soon?